supervisor/api/ingress.py

```diff
@@ -92,7 +92,7 @@ def _extract_addon(self, request: web.Request) -> Addon:
 
         return addon
 
-    def _create_url(self, addon: Addon, path: str) -> str:
+    async def _async_create_url(self, addon: Addon, path: str) -> str:
```

Comment:
```suggestion
    def _create_url(self, addon: Addon, path: str) -> str:
```
It's even not async

Reply:
Right, that came from an earlier implementation where it had to be. Will fix 👍